Skype/Trusted-Application-API/SDK/Tests/ClientModel/AudioVideoFlow.cs

```diff
@@ -197,6 +211,33 @@ public void StatePropertyShouldGetUpdatedOnEvent()
         }
 
         [TestMethod]
+        public async Task StopPromptsShouldReturnOnlyOnPromptCompletedEvent()
```

Comment:
Add following test cases as well:
1. StopPromptAsync called when no prompt is being played
2. StopPromptAsync called when multiple prompts are in the playing queue